backend/app.py

```diff
@@ -153,11 +156,11 @@ def addUserToSquad(invite_link, email):
 @login_required # If you want to test this endpoint w/o requiring auth (i.e. Postman) comment this out
 def createEvent():
     content = request.get_json()
-    ok, err = validateArgsInRequest(content, "auth_hash", "title",
-                                    "description", "start_time", "end_time", "address", "squad_id")
+    ok, err = validateArgsInRequest(content, "email", "title", "emoji",
```

Comment:
for fields not provided by user, mobile sends "null", so validation will still pass